tests/test_unit.py

```diff
@@ -2084,5 +2086,77 @@ def test_custom_regex_escape(self):
                 assert result == output
 
 
+class KafkaAvailabilityTests(TestCase):
+    def setUp(self,):
+        super().setUp()
+        self.config = Config(RuntimeEnvironment.TEST)
+
+    def test_happy_path(self):
+        real = host_kafka
+        real._any_bootstrap_server_connects = MagicMock()
+        real._any_bootstrap_server_connects.return_value = True
+
+        available = real.kafka_available()
+
+        assert available is True
+        real._any_bootstrap_server_connects.assert_called_once_with(mock.ANY, None)
```

Comment:
This is more of an issue of readability... but why did you make a variable named `real` which is set to the whole `host_kafka` class? `available` also seems unnecessary, since it's only read once. It would be more concise and IMO easier to understand without the extra vars:

```suggestion
        host_kafka._any_bootstrap_server_connects = MagicMock()
        host_kafka._any_bootstrap_server_connects.return_value = True

        assert host_kafka.kafka_available()
        host_kafka._any_bootstrap_server_connects.assert_called_once_with(mock.ANY, None)
```